testdata/ppc-expected-profiles/profile2.json

```diff
@@ -2,6 +2,6 @@
   "must-gather-dir-path": "must-gather.sno",
   "mcp-name": "master",
   "reserved-cpu-count": 1,
-  "rt-kernel": false
+  "rt-kernel": false,
+  "user-level-networking": false
```

Comment:
I see we have added `user-level-networking` argument here. Just to demonstrate exactly what this PR does, lets add a test-case where user-level-networking is not passed as an argument and we should be able to the expected profile where `userLevelNetworking` field does not exist in the generated performance profile.

Reply:
Thanks for the review, I actually test the exact case you mentioned with the other pair: profile1.{yaml,json}
I removed the "net" field from that profile2.yaml so it can check for it.